src/services/authService.js

```diff
@@ -36,7 +36,7 @@ import {firebaseAuth} from "../firebase"
       dispatch(login())
       cfaSignIn("google.com").subscribe((user)=>{
        setuserData(user.displayName,user.email,user.photoURL);
-       
+       user.sendEmailVerification();
```

Comment:
This is not the way to add a new firebase signIN method. You are actually adding this method to the method that is working for google authentication.

references: https://firebase.google.com/docs/auth/web/password-auth
create a new method to handle email-password auth

Reply:
Ohh! sorry i just re-read it now i know i also need to enable login with email and password other than google sign-in ,right.

Reply:
yes, you are right.

Reply:
hey are you using django in it ,I do not know django so i tried but i  think i am unable to do it. Sorry for wasting time.

Reply:
No, we are not using django. It should be written in ReactJS

Reply:
ok i will try again ,but i think @account/ in accountActions is django when i googled it.

Reply:
A quick discord chat?